tests/example_policies/testSemanticDiff/with-different-topologies-scheme.yaml

```diff
@@ -0,0 +1,67 @@
+namespaceList: ../../example_podlist/ns_list.json
+podList: ../../example_podlist/pods_list.json
+
+# testing with networkPolicyLists (for backward compatibility)
+# can use only global namespaceList and podList resources
+networkPolicyLists:
+  - name: no_policy
+    networkPolicyList: []
+    expected_warnings: 0
+
+  - name: policy
+    namespaceList: ../tests/example_podlist/ns_list.json
```

Comment:
This is currently not allowed. Does that mean that `networkPolicyLists` and `networkConfigList` are synonyms?

Reply:
Yes, good point. Therefore I suggest to clean up and remove deprecated `networkPolicyLists`

Reply:
Deprecated networkPolicyLists is removed